india_compliance/patches/v14/update_item_gst_details_and_gst_trearment_in_bill_of_entry.py

```diff
@@ -88,8 +90,16 @@ def update_gst_details(company, doctype, docs):
 
 
 def get_taxes_for_docs(docs, doctype):
-    taxes_doctype = "Bill of Entry Taxes"
-    taxes = frappe.qb.DocType(taxes_doctype)
+    boe_taxes = frappe.qb.DocType("Bill of Entry Taxes")
+    ic_taxes = frappe.qb.DocType("India Compliance Taxes and Charges")
+
+    return (
+        get_taxes_query(docs, doctype, boe_taxes)
```

Comment:
This may also be used where this table doesn't exist. I guess we should make provision for the same.